### DISCUSSION TITLE
Python

### DISCUSSION THREAD

**Discussion by Morphin16** [Rapptz/discord.py#8237] [General]
Begginners guide

**Comment by mesub7** [comment]
Hello,

Welcome to discord.py! 

If you're looking for how to use the library then there are a range of options.

[The documentation](https://discordpy.readthedocs.io/en/stable/) provides a rich source of information on using the library - there's also a quickstart guide there.

In addition to that, there are also various [examples](https://github.com/Rapptz/discord.py/tree/master/examples) to understand different elements in practice.


Alternatively, [this guide](https://realpython.com/how-to-make-a-discord-bot-python/) can also give some grounding on the library. 


It must be said that discord.py isn't a beginner's library; if your knowledge of python is still quite basic, if may be worth looking into doing smaller projects first to build your confidence.


Good luck!
